webtool/frontend/src/components/intro/IntroText.tsx

```diff
@@ -1,63 +1,94 @@
-import {FunctionComponent, PropsWithChildren} from "react"
+import {ComponentProps, FunctionComponent, PropsWithChildren} from "react"
 import {ExternalLink} from "../ExternalLink.tsx"
+import {Grid, Heading} from "@radix-ui/themes"
+import {css} from "@emotion/react"
+import {
+    local4localDarkOrange,
+    local4localLightBlue,
+    local4localLightGreen,
+    local4localLightOrange
+} from "../../colors.ts"
 
-const wikiLink = "https://local4local.notion.site/Hoe-maak-ik-een-grove-berekening-van-de-gelijktijdigheid-van-mijn-energiegemeenschap-17a0306d63c2805ba3c8d60695d662b5";
+const wikiLink = "https://local4local.notion.site/Hoe-maak-ik-een-grove-berekening-van-de-gelijktijdigheid-van-mijn-energiegemeenschap-17a0306d63c2805ba3c8d60695d662b5"
 
-const H2: FunctionComponent<PropsWithChildren> = ({children}) => (
-    <h2 style={{paddingTop: "1.5rem"}}>{children}</h2>
+const H2: FunctionComponent<PropsWithChildren & ComponentProps<typeof Heading>> = ({children, style}) => (
```

Comment:
PropsWithChildren can be removed.

(aside, the usual way to use it would be `PropsWithChildren<ComponentProps<typeof Heading>>`)